src/packages/excalidraw/CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ Please add the latest change on the top under the correct section.
 
 #### Features
 
+- ##### BREAKING CHANGE
```

Comment:
I've done that first, but it turned out not very useful. The added APIs and breaking changes go hand in hand.

Reply:
Yes they do go hand in hand but what I mean here is since "Features" is the main category so let's add what are the features first and then mention breaking changes (breaking changes are part of migration so better to first explain what are features and then explain breaking changes as thats more intuitive and expected order)

Reply:
updated